packages/manager/apps/cda/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@ovh-ux/manager-cda-app",
+  "version": "0.0.0",
+  "private": true,
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ovh/manager.git",
+    "directory": "packages/manager/apps/cda"
+  },
+  "license": "BSD-3-Clause",
+  "author": "OVH SAS",
+  "scripts": {
+    "build": "yarn build:eu && yarn build:ca && yarn build:us",
+    "build:ca": "webpack --env.production --env.region='CA' && mv dist dist-CA",
+    "build:eu": "webpack --env.production && mv dist dist-EU",
+    "build:us": "webpack --env.production --env.region='US' && mv dist dist-US",
+    "dev": "webpack-dev-server",
+    "dev:watch": "yarn run dev",
+    "start": "lerna exec --stream --scope='@ovh-ux/manager-cda-app' --include-dependencies -- npm run build --if-present",
+    "start:dev": "lerna exec --stream --scope='@ovh-ux/manager-cda-app' --include-dependencies -- npm run dev --if-present",
+    "start:watch": "lerna exec --stream --parallel --scope='@ovh-ux/manager-cda-app' --include-dependencies -- npm run dev:watch --if-present"
+  },
+  "dependencies": {
+    "@ovh-ux/manager-cda": "^0.0.0 || ^0.1.0",
+    "@ovh-ux/manager-config": "^0.4.0",
+    "@ovh-ux/manager-core": "^8.0.0 || ^9.0.0",
+    "@ovh-ux/ng-at-internet": "^5.0.0",
+    "@ovh-ux/ng-at-internet-ui-router-plugin": "^3.0.0",
+    "@ovh-ux/ng-ovh-api-wrappers": "^4.0.7",
+    "@ovh-ux/ng-ovh-cloud-universe-components": "^1.4.0",
+    "@ovh-ux/ng-ovh-doc-url": "^1.0.0",
+    "@ovh-ux/ng-ovh-form-flat": "^4.0.3",
+    "@ovh-ux/ng-ovh-http": "^4.0.6",
+    "@ovh-ux/ng-ovh-proxy-request": "^1.0.1",
+    "@ovh-ux/ng-ovh-request-tagger": "^1.1.0",
+    "@ovh-ux/ng-ovh-sidebar-menu": "^8.5.1",
+    "@ovh-ux/ng-ovh-sso-auth": "^4.2.3",
+    "@ovh-ux/ng-ovh-swimming-poll": "^4.0.2",
+    "@ovh-ux/ng-ui-router-layout": "^3.2.0",
+    "@ovh-ux/ui-kit": "^4.1.12",
+    "@uirouter/angularjs": "^1.0.23",
+    "angular": "^1.7.5",
+    "angular-aria": "^1.7.8",
+    "angular-chart.js": "^1.1.1",
+    "angular-cookies": "^1.7.8",
+    "angular-dynamic-locale": "^0.1.37",
+    "angular-i18n": "^1.7.8",
+    "angular-resource": "^1.7.8",
+    "angular-sanitize": "^1.7.8",
+    "angular-translate": "^2.18.1",
+    "angular-translate-loader-pluggable": "^1.3.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "bootstrap": "~3.3.7",
+    "chart.js": "chartjs/Chart.js#^2.0",
+    "flatpickr": "~4.5.2",
+    "jquery": "^2.1.3",
+    "moment": "^2.19",
+    "ovh-api-services": "^9.41.1",
+    "ovh-ui-angular": "^3.15.1",
```

Comment:
We don't use `ovh-ui-angular` anymore, so I think we should rebase/update this PR.